tools/c7n_azure/c7n_azure/resources/cosmos_db.py

```diff
@@ -42,7 +64,7 @@ class CosmosDB(ArmResourceManager):
 
     class resource_type(ArmResourceManager.resource_type):
         service = 'azure.mgmt.cosmosdb'
-        client = 'CosmosDB'
+        client = 'CosmosDB'  # type: azure.mgmt.cosmosdb.CosmosDB
```

Comment:
(minor) I don't think we have been checking in the typing hints

Reply:
We should!